src/main/webapp/src/components/Basket.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import axios from "axios";
+import {Button} from "antd";
+import {DeleteFilled, ShoppingOutlined} from "@ant-design/icons";
+import '../styles/Basket.css'
+import '../styles/NotFoundPage.css'
+import {Link} from "react-router-dom";
+
+class Basket extends React.Component {
+
+    componentDidMount() {
+        axios.get('/api/basket').then();
+    };
+
+    clearBasket = () => {
+        axios.delete("/api/basket/clear").then(response => {
+            this.props.setBasket(undefined);
+        });
+    }
+
+    removeItems = (id) => {
+        axios.delete(`/api/basket/removeitem/${id}`).then(response => {
+                axios.get('/api/basket').then(response => {
+                    const basket = response.data;
+                    this.props.setBasket(basket);
+                });
+            }
+        );
+
+    }
```

Comment:
all these methods better also move to App.js as basket state is managed there

Reply:
+